test/tests/reporter/GriddedDataReporter/tests

```diff
@@ -0,0 +1,9 @@
+[Tests]
+requirement = "This tests that a reporter is able to read in a gridded data file and produce the appropriate grid from it that can be used in PiecewiseMultiInterpolation like functions."
```

Comment:
get rid of the detail comment and move the requirement comment inside the gReporterexo block.  Also rename this test to xyGrid

Reply:
done